### PR TITLE
[script.kodi.android.update] 1.0.4

### PR DESCRIPTION
**Pull request by Lunatixz**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.kodi.android.update/resources/language/resource.language.en_gb/strings.po [hunk 1]
```diff
@@ -45,9 +45,29 @@ msgid "Redownload"
...
+msgctxt "#30009"
+msgid "Plugin disabled"
+msgstr ""
+
+msgctxt "#30010"
+msgid "Unknown Android version"
+msgstr ""
+
+msgctxt "#30011"
+msgid "Android v.%s.0.0 is incompatible with Kodi,"
```
Comment (MartijnKaijser) [inline script.kodi.android.update/resources/language/resource.language.en_gb/strings.po]: Is incompatible with new Kodi versions

File: script.kodi.android.update/service.py [hunk 2]
```diff
@@ -44,8 +44,8 @@ def __init__(self):
                 log('Service, removing last download')
                 xbmcvfs.delete(self.lastPath)
                 xbmcgui.Dialog().notification(ADDON_NAME, LANGUAGE(30007), ICON, 4000)
-        except Exception as e:
+        except Exception as e: 
             log("Service Failed! " + str(e), xbmc.LOGERROR)
         
-if __name__ == '__main__':
+if __name__ == '__main__':
```
Comment (MartijnKaijser) [inline script.kodi.android.update/service.py]: Some redundant white spaces

**Comment by MartijnKaijser** [comment]
Should be added to Jarvis branch. Mind the xbmc.python dependency

**Comment by Lunatixz** [comment]
Oh you want the whole script moved to jarvis? Figured you wanted the ability to manually install in jarvis. 

I'll have to review the code in order to make sure I'm not using Krypton exclusive API methods.

**Pull request closed without merging**